source/transfer_list.rst

```diff
@@ -98,6 +98,35 @@ changes will be backwards-compatible to older readers.
      - 0xc
      - The maximum number of bytes that the TL can occupy. Any entry producer must check if there is sufficient space before adding an entry to the list. Firmware can resize and/or relocate the TL and update this field accordingly, provided that the TL requirements are respected. This field must be a multiple of 8.
 
+   * - flags
+     - 0x4
+     - 0x10
+     - Flags word. See below for contents.
+
+   * - reserved
+     - 0x4
+     - 0x14
+     - Reserved word. Must be set to 0 or ignored.
+
+
+TL Flags
```

Comment:
Do we have the need for the flags field today?
having whole-TL flags is valuable, and we'll certainly need this at some point.
Until we have a concrete need for the flag, is it best to leave the field as "reserved"? -- The reserved field would effectively be 8-bytes wide.

If we do leave it entirely as reserved, we could comment the src saying that we intended to use part of the field for flags.